.circleci/config.yml

```diff
@@ -1,14 +1,22 @@
 version: 2.1
 
 commands:
-  set_up_environment:
+  install_nix:
```

Comment:
choose consistent command names - verb phrases (install nix) or noun phrases (submodule update)

Reply:
That's fair.  The existing commands seem to all be verb phrases, so I'll go with that.